src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -102,7 +103,7 @@ public void should_read_from_blob_storage_and_save_metadata_in_database_when_zip
     public void should_process_other_zip_files_if_previous_zip_fails_to_process() throws Exception {
         // given
         uploadZipToBlobStore("3_24-06-2018-00-00-00.zip"); //Zip with only pdf without metadata
-        uploadZipToBlobStore("4_24-06-2018-00-00-00.zip"); //Zip with pdf and metadata
+        uploadZipToBlobStore("7_24-06-2018-00-00-00.zip"); //Zip with pdf and metadata
```

Comment:
did this have to change? ie, was `4_24-06-2018-00-00-00.zip` invalid?

Reply:
Zip file `4_...` was wrong from the very beginning. Since we did not check integrity before uploading, the error never appeared and mock always provided correct response